browser/js/create-game/create-game.controller.js

```diff
@@ -41,5 +42,16 @@ app.controller('CreateGameCtrl', function($scope, $mdDialog){
             scope: $scope
         });
     }
+    $scope.getMatches = function() {
+        UserFactory.autocomplete($scope.searchText)
+        .then(users=>$scope.foundMatches = users)
+        .catch(err=>$log.error)
+
+    }
+
+    $scope.create = function(){
+        GameFactory.createGame($scope.comm)
```

Comment:
Maybe I'm missing it but GameFactory doesn't seem to be injected into this controller